src/index.html

```diff
@@ -5,9 +5,371 @@
+          <div class="reviews__card card">
+            <img
+              src="images/people/Ellipse1.png"
+              alt="Leslie"
+              class="card__img"
+            >
+            <h5 class="card__name">
+              Leslie McKinney
+            </h5>
+            <p class="card__position">CEO</p>
+            <p class="card__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+
+          <div class="reviews__card card">
+            <img
+              src="images/people/Ellipse2.png"
+              alt="Leslie"
+              class="card__img"
+            >
+            <h5 class="card__name">
+              Scarlett Hawkins
+            </h5>
+            <p class="card__position">Senior Vice President</p>
+            <p class="card__text">
+              My soul is illuminated by an unearthly joy, as these
+               beautiful spring morning, which I enjoy with all my heart
+            </p>
+          </div>
+
+          <div class="reviews__card card">
+            <img
+              src="images/people/Ellipse3.png"
+              alt="Leslie"
+              class="card__img"
+            >
+            <h5 class="card__name">
+              Max Henry
+            </h5>
+            <p class="card__position">Chief Design Officer</p>
+            <p class="card__text">
+              When my lovely valley of steam rises and half-day
+               sun is above an impermeable
+            </p>
+          </div>
+        </div>
+      </section>
+      <!--Contacts-->
+      <section class="contacts page__contacts section" id="contacts">
+        <h2 class="contacts__name name">Contact us</h2>
+        <img
+          src="images/crown.svg"
+          alt="logo"
+          class="contacts__logo"
+        >
+        <div class="contacts__main">
+          <div class="contacts__details">
+            <p class="contacts__label">Phone</p>
+            <p class="contacts__phone">+1 234 5555-55-55</p>
+            <p class="contacts__label">Email</p>
+            <p class="contacts__email">hello@miami.com</p>
+          </div>
+          <div class="contacts__details">
+            <p class="contacts__label">Address</p>
+            <p class="contacts__address">
+              400 first ave. n. <br> suite 700 <br> Minneapolis, MN <br>55401
+            </p>
+          </div>
+          <div class="contacts__form form">
+            <form
+              action="aionovaionova@gmail.com"
+              method="POST"
+              class=" consultation__form form"
+            >
+              <input
+                type="text"
+                placeholder="Name"
+                required
+                name="name"
+                class="form__input"
+              >
+              <input
+                type="email"
+                placeholder="Email"
+                name="email"
+                class="form__input"
+              >
```

Comment:
Класно що в імені є але думаю до пошти та номера в наступній формі також варто додати, бо ще без імені клієнту можна відповісти а от без контактів не знаю!) Я також спочатку про це не подумав..
```suggestion
              <input
                type="email"
                placeholder="Email"
                name="email"
                class="form__input"
                required
              >
```
[Тут вже писали..](https://mateacademy.slack.com/archives/G018RCJTGH2/p1600005323236300)

Reply:
add required